web/impact/impact/tests/test_algolia_api_key_view.py

```diff
@@ -159,32 +205,32 @@ def test_alumni_user_who_is_also_finalist_sees_mentors_of_both_programs(
                     self.assertIn(program.name, response_data["filters"])
                 self.assertIn(other_program.name, response_data["filters"])
 
-    def test_non_participant_user_sees_all_confirmed_mentors(self):
-        named_group = NamedGroupFactory()
-        programs = ProgramFactory.create_batch(
-            5,
-            mentor_program_group=named_group,
-            program_status=ACTIVE_PROGRAM_STATUS)
+    def test_superuser_employee_sees_all_mentors(self):
+        user = self._create_expert()
+        user.is_superuser = True
+        user.save()
 
-        program = programs[0]
-        user = self._create_user_with_role_grant(program,
-                                                 UserRole.DESIRED_MENTOR)
         with self.settings(
                 ALGOLIA_APPLICATION_ID='test',
                 ALGOLIA_API_KEY='test'):
             with self.login(email=user.email):
                 response = self.client.get(self.url)
                 response_data = json.loads(response.content)
 
-                self.assertIn(IS_CONFIRMED_MENTOR_FILTER,
-                              response_data["filters"])
+                self.assertEqual(response_data["filters"], [])
 
     def _create_entrepreneur(self):
         ent_user = EntrepreneurFactory()
         ent_user.set_password("password")
         ent_user.save()
         return ent_user
 
+    def _create_expert(self):
```

Comment:
It's not clear why this is an instance method, since it does not use `self`.

This duplicates the previous function, differing only in the choice of factory. Can refactor as 
```
def _create_expert():
  return _create_user(ExpertFactory)

def _create_entrepreneur():
  return _create_user(EntrepreneurFactory)

def _create_user(factory):
  user = factory()
  user.set_password("password")
  user.save()
  return user
```

Reply:
Uuuu nice, very clean. I like it!